`make providers...` is broken

**Describe the bug**
```
<input>: invalid character NUL
<input>: invalid character NUL
panic: runtime error: index out of range [1025] with length 1025

goroutine 1 [running]:
text/scanner.(*Scanner).next(0x0?)
        /home/linuxbrew/.linuxbrew/Cellar/go/1.21.3/libexec/src/text/scanner/scanner.go:219 +0x4e5
text/scanner.(*Scanner).Scan(0xc0001ff400)
        /home/linuxbrew/.linuxbrew/Cellar/go/1.21.3/libexec/src/text/scanner/scanner.go:733 +0x331
github.com/alecthomas/participle/lexer.(*textScannerLexer).Next(0xc0001f2570)
        /home/christian/go/pkg/mod/github.com/alecthomas/participle@v0.7.1/lexer/text_scanner.go:91 +0x54
github.com/alecthomas/participle/lexer.Upgrade({0x94d260, 0xc0001f2570})
        /home/christian/go/pkg/mod/github.com/alecthomas/participle@v0.7.1/lexer/peek.go:14 +0x8e
github.com/alecthomas/participle.(*Parser).Parse(0xc00007c3c0, {0x94d240?, 0xc0001ee660?}, {0x7fbac0, 0xc000152d80}, {0x0, 0x0, 0x0})
        /home/christian/go/pkg/mod/github.com/alecthomas/participle@v0.7.1/parser.go:167 +0x58
go.mondoo.com/cnquery/v9/providers-sdk/v1/lr.Parse({0xc0001b0000, 0x8310})
        /home/christian/workspace/mondoo/github.com/cnquery/providers-sdk/v1/lr/lr.go:227 +0x172
go.mondoo.com/cnquery/v9/providers-sdk/v1/lr.Resolve({0x7ffc0cb0c9c3, 0x1c}, 0x8dc1b0)
        /home/christian/workspace/mondoo/github.com/cnquery/providers-sdk/v1/lr/resolve.go:21 +0x94
go.mondoo.com/cnquery/v9/providers-sdk/v1/lr/cli/cmd.glob..func3(0xc000188300?, {0xc00019a2d0, 0x1, 0x89a06b?})
        /home/christian/workspace/mondoo/github.com/cnquery/providers-sdk/v1/lr/cli/cmd/go.go:33 +0x125
github.com/spf13/cobra.(*Command).execute(0xc7bce0, {0xc00019a240, 0x3, 0x3})
        /home/christian/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x863
github.com/spf13/cobra.(*Command).ExecuteC(0xc7c580)
        /home/christian/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
        /home/christian/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
go.mondoo.com/cnquery/v9/providers-sdk/v1/lr/cli/cmd.Execute()
        /home/christian/workspace/mondoo/github.com/cnquery/providers-sdk/v1/lr/cli/cmd/root.go:31 +0x1a
main.main()
        /home/christian/workspace/mondoo/github.com/cnquery/providers-sdk/v1/lr/cli/main.go:17 +0xf
make: *** [Makefile:200: providers/build/os] Error 2
```

I added providers to the `providers.yaml` file and called `make providers/config`. That resulted in the above panic.

One of the commits between current main and 4dabf6500a30825ef72fcca82778a540e8b57988 broke the lexer.

**Expected behavior**
This should still work.

--- a/providers/github/go.mod
+++ b/providers/github/go.mod
@@ -33,7 +33,7 @@ require (
 	github.com/OpenPeeDeeP/depguard/v2 v2.1.0 // indirect
 	github.com/ProtonMail/go-crypto v0.0.0-20230923063757-afb1ddc0824c // indirect
 	github.com/StackExchange/wmi v1.2.1 // indirect
-	github.com/alecthomas/participle v0.7.1 // indirect
+	github.com/alecthomas/participle v0.3.0 // indirect
 	github.com/alecthomas/participle/v2 v2.1.0 // indirect
 	github.com/alexkohler/nakedret/v2 v2.0.2 // indirect
 	github.com/alexkohler/prealloc v1.0.0 // indirect

--- a/providers/google-workspace/go.mod
+++ b/providers/google-workspace/go.mod
@@ -36,7 +36,7 @@ require (
 	github.com/OpenPeeDeeP/depguard/v2 v2.1.0 // indirect
 	github.com/ProtonMail/go-crypto v0.0.0-20230923063757-afb1ddc0824c // indirect
 	github.com/StackExchange/wmi v1.2.1 // indirect
-	github.com/alecthomas/participle v0.7.1 // indirect
+	github.com/alecthomas/participle v0.3.0 // indirect
 	github.com/alecthomas/participle/v2 v2.1.0 // indirect
 	github.com/alexkohler/nakedret/v2 v2.0.2 // indirect
 	github.com/alexkohler/prealloc v1.0.0 // indirect

